### PR TITLE
Remove nth node from linkedlist

### PR DESCRIPTION
**Pull request by codewithnick**
## Have you read the Contribution Guidelines?

yes

## Description

program to delete nth node from linked list
I have used a dynamic linked list and implemented a function to remove nth node from linked list
where n is given  by user

## Checklist
- [x] I've read the contribution guidelines.
- [x] I've checked the issue list before deciding what to submit.
- [x] I've edited the README.md and link to my code.
 
 
## Related Issues (Mention Issue no. like #xxx)
#19 
Fixes Issue 
<br/>
(Write your answer above.)

### CODE REVIEW

File: Python/remove_nth_node_from_linkedlist.py [hunk 1]
```diff
@@ -0,0 +1,148 @@
...
+    elif(op==2):
+        linkedlist.display()
+    elif (op==3):
+        n=int(input('enter the number of node you want to delete'))
+        linkedlist.DeleteNthNode(n-1)
+        print("new linked list")
+        linkedlist.display()
+
+
+##
```
Comment (sukritishah15) [inline Python/remove_nth_node_from_linkedlist.py:83]: PUT ALL THE LINES STARTING HERE IN A SINGLE MULTILINE COMMENT.
Add time and space compexity.

File: Python/stack.py [hunk 2]
```diff
@@ -0,0 +1,143 @@
...
+        s1.isEmpty()
+        
+    else:
+        print("Invalid option.")
+
+    print("Enter option that you want to perform")
+    op=int(input("1:push 2:pop 3:display 4:peek 5:length 6:isEmpty : "))
+    print()
+
+
```
Comment (sukritishah15) [inline Python/stack.py:80]: PUT ALL THE LINES STARTING HERE IN A SINGLE MULTILINE COMMENT.
Add time and space compexity.

**Comment by sukritishah15** [comment]
I'm closing this due to SINGLE LINE COMMENTS.
Create a new PR.

File: Python/remove_nth_node_from_linkedlist.py (hunk 1 above)
Comment (codewithnick) [reply Python/remove_nth_node_from_linkedlist.py:83]: can I take all my files in a new branch and make a new pr
actually I had made 4 different prs for four files for hacktoberfest
i have the final files ready should i put them all under one branch and merge it 
or make differenr prs for different files
please tell me

**Pull request closed without merging**